Tasks/DeployAzureResourceGroup/Utility.ps1

```diff
@@ -58,6 +58,22 @@ function Write-TaskSpecificTelemetry
     Write-Telemetry "$codeKey" "94A74903-F93F-4075-884F-DC11F34058B4"
 }
 
+function Get-AzureCmdletsVersion
+{
+    $module = Get-Module AzureRM -ListAvailable
+    if($module)
+    {
+        return ($module).Version
+    }
+    return (Get-Module Azure -ListAvailable).Version
```

Comment:
I assume you are closing this with @niadak if the check should involve AzureRm.Profile.